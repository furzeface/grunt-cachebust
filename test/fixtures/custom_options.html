<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Index</title>
  <link rel="stylesheet" href="styles/main.css" />
  <link rel="stylesheet" href="http://cdn.com/styles/main.css" />
  <script src="scripts/main.js"></script>
</head>
<body>
  <script src="scripts/moar.js"></script>
  <script src="http://best-cdn-ever.com/scripts/evenMoar.js"></script>
</body>
</html>
