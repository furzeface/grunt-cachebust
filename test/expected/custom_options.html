<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Index</title>
  <link rel="stylesheet" href="styles/main.css?t=1415128382750" />
  <link rel="stylesheet" href="http://cdn.com/styles/main.css" />
  <script src="scripts/main.js?t=1415128382750"></script>
</head>
<body>
  <script src="scripts/moar.js?t=1415128382750"></script>
  <script src="http://best-cdn-ever.com/scripts/evenMoar.js"></script>
</body>
</html>
